jolie/src/main/java/jolie/Interpreter.java

```diff
@@ -1744,5 +1748,9 @@ public File programDirectory() {
 		public void clear() {
 			jolieClassLoader = null;
 		}
+
+		public Optional< File > getParametersFilePath() {
```

Comment:
Just `parametersPath` is fine as method name. (And return an `Optional< Path >`.)